setup.py

```diff
@@ -44,26 +44,26 @@
     "accelerate==0.23.0",
     "bitsandbytes==0.41.1",
```

Comment:
I think there may be a bug with this version. I saw this on twitter:
https://twitter.com/Tim_Dettmers/status/1722312834003124698

Reply:
Bumped deps here to be future proof (also included the PEFT patch release): https://github.com/huggingface/alignment-handbook/pull/11/commits/0af801199336e59a338c6f2d54c8f4a7bffb9eb1